shell/components/form/Select.vue

```diff
@@ -91,8 +92,18 @@ export default {
       type:    Boolean,
       default: false
     },
+    // at the time of this code update there is no option
+    // in the vue-select component to add a custom aria-label
+    // the next best thing is to render a screen-reader
+    // hidden element and connect it via ID
+    hiddenLabel: {
```

Comment:
nitpick: I think that the name of this prop can be made more clear if we communicate its intended use with something along the lines of `screenReaderLabel` or `srLabel`

Reply:
Closing as it's not applicable after last commit